data/json/items/armor/boots.json

```diff
@@ -2286,9 +2285,10 @@
     "price": 5012,
     "price_postapoc": 2000,
     "to_hit": -1,
+    "bashing": 4,
     "symbol": "[",
     "color": "light_red",
-    "flags": [ "VARSIZE", "STURDY", "OUTER" ],
+    "flags": [ "VARSIZE", "STURDY", "OUTER", "DURABLE_MELEE" ],
```

Comment:
this shouldnt have durable melee since the metal is poor quality

Reply:
fair enough